docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
+* preview running experiment
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
 
-.. code-block:: bash
+.. tip::
 
-    pip install neptune-client
+    Check this public project with example experiments: |project|.
 
+    You can also go straight to the |exp|.
 
-Create the **LightningModule**
-------------------------------
+Quickstart
+----------
 
-Nothing changes here, just your typical ``LightningModule`` structure.
+Introduction
+^^^^^^^^^^^^
+This quickstart will show you how to use Neptune with your existing PyTorch Lightning code. You have three options:
 
-.. code-block:: python3
+1. Follow through this quickstart for detailed explanations,
+2. Open starter code in Colab Notebook (badge-link below) and run it as a "`neptuner`" user - zero setup, it just works.
+3. View starter code as Python script on |script|.
 
-    class CoolSystem(pl.LightningModule):
+|Run on Colab|
 
-        def __init__(self):
-            ...
+Before you start
+^^^^^^^^^^^^^^^^
 
-        def forward(self, x):
-            ...
+**Prerequisites**
 
-        def training_step(self, batch, batch_idx):
-            ...
+* Python 3,
+* Neptune account |register|,
+* Minimal familiarity with the PyTorch Lightning.
 
-        def configure_optimizers(self):
-            ...
+**Supported version**
 
-        @pl.data_loader
-        def train_dataloader(self):
-            ...
+* ``pytorch-lightning>=0.9.0``
+* ``neptune-client>=0.4.105``
 
-        @pl.data_loader
-        def val_dataloader(self):
-            ...
+Installation
+^^^^^^^^^^^^
+**Install neptune-client**
 
-Create the **NeptuneLogger** with all the information you want to track
------------------------------------------------------------------------
-.. code-block:: python3
+From PyPI:
 
-    from pytorch_lightning.loggers.neptune import NeptuneLogger
+.. code-block:: bash
 
-    neptune_logger = NeptuneLogger(
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        experiment_name="default",  # Optional,
-        params={"max_epochs": 10,
-                "batch_size": 32},  # Optional,
-        tags=["pytorch-lightning", "mlp"]  # Optional,
-    )
-
-Create the **Trainer** and pass **neptune_logger** to logger
-------------------------------------------------------------
-.. code-block:: python3
+    pip install neptune-client -U
 
-    from pytorch_lightning import Trainer
+From Conda:
 
-    model = CoolSystem()
-    trainer = Trainer(max_epochs=10, logger=neptune_logger)
+.. code-block:: bash
 
-    trainer.fit(model)
+    conda install neptune-client -c conda-forge
 
-Log additional information after the **.fit** loop ends
--------------------------------------------------------
+**Install PyTorch Lightning and torchvision**
```

Comment:
This should be in pre-requisites with a link to their documentation on how to install I believe -> it is not Neptune related.

Reply:
corrected.